modules/openapi-generator/src/main/resources/aspnetcore/2.1/Startup.mustache

```diff
@@ -42,11 +42,11 @@ namespace {{packageName}}
         {
             // Add framework services.
             services
-                .AddMvc()
+                .AddMvc({{^useEndpointRouting}}opts => opts.EnableEndpointRouting = false{{/useEndpointRouting}})
```

Comment:
I am wondering if for a web api it would make more sense to just use MvcCore

Reply:
It probably might - for now the option to turn off the Swashbuckle support (useSwashbuckle) is there - eventually the default could be flipped. It might be a breaking change for people aleady using it so avoided doing that. 

I have tried to keep the old behaviour (particularly for 2.2 and earlier) unchanged as much as possible.

Reply:
Variable name was also wrong  should have been useDefaultRouting